src/fromager/commands/lint_requirements.py

```diff
@@ -0,0 +1,48 @@
+import glob
+import itertools
+import logging
+import sys
+
+import click
+from packaging.requirements import InvalidRequirement, Requirement
+
+from fromager import requirements_file
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument("input_files_path", nargs=-1, type=click.Path(exists=False))
+def lint_requirements(input_files_path: str) -> None:
+    """
+    Command to lint the constraints.txt and requirements.txt files
+    This command takes a single wildcard path string for constraints.txt and requirements.txt.
+    It checks the formatting of these files and reports issues if found.
+    """
+    # Exit if user does not provide wildcard paths
+    if not input_files_path:
+        logger.error("path for requirements.txt or constraints.txt is missing")
+        sys.exit(1)
+
+    # Get all the files to check in a list
+    files_to_check = list(
+        itertools.chain.from_iterable(glob.glob(path) for path in input_files_path)
```

Comment:
By the time the linter is invoked, the shell will have expanded the wildcard for us, so we don't need this glob call.